infrastructure/index.ts

```diff
@@ -312,6 +312,11 @@ const cdn = new aws.cloudfront.Distribution(
     {
         protect: true,
         dependsOn: [ websiteBucket, websiteLogsBucket ],
+        aliases:  [
+            config.websiteDomain,
+            config.targetDomain,
+            config.redirectDomain || "",    // redirectDomain may be undefined.
+        ].filter(domain => domain !== ""),
```

Comment:
The "array literal" immediately passed to `.filter(` seems kinda lame, IMHO.

Can we just declare a variable elsewhere? That would also give us the opportunity to better explain the (also lame) reason why we have up to 3 different aliases. (* I can call this all lame, since it's kinda-sorta my fault we have those three 😭 )

```
const domainAliases = [];
// website domain does X. It is important because of Y
domainAliases.push(config.websiteDomain);
//  target domain does X. This is a lame hack, but necessary because of Y.
domainAliases.push(config.targetDomain);
// Optionally set the redirect domain, you know, because of X.
if (config.redirectDomain) {
     domainAliases.push(config.redirectDomain);
}
```

What do you think?

Reply:
Sure, that works. 👍